teamsyncd/teamsync.cpp

```diff
@@ -140,14 +144,23 @@ int TeamSync::TeamPortSync::onPortChange(bool isInit)
         {
             string key = m_lagName;
             key += ":";
+
             uint32_t ifindex = team_get_port_ifindex(port);
             char ifname[MAX_IFNAME + 1] = {0};
-            key += team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME);
+
+            team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME);
+            /* netdev interface is missing */
+            if (ifname[0] == 0)
```

Comment:
what will be the ifindex then in this case, is ifindex valid or not? can we use that? 

Also since the underlying port is absent, should we remove the the port from the lag? I do not see you call the del logic there.

---
In reply to: [83899450](https://github.com/Azure/sonic-swss/pull/109#discussion_r83899450) [](ancestors = 83899450)